client/src/lit/play/console-utils.js

```diff
@@ -17,6 +18,17 @@ export function formatArray(input) {
       output += "Array [";
       output += formatArray(input[i]);
       output += "]";
+    } else if (!input.hasOwnProperty(i)) {
+      let emptyCount = 1;
+      while (i + 1 < l && !input.hasOwnProperty(i + 1)) {
+        emptyCount++;
+        i++;
+      }
+      if (emptyCount === 1) {
+        output += "<empty>";
```

Comment:
```suggestion
        output += "<${emptyCount} empty slot>";
```
Empty slot is shown as `<1 empty slot>` in Firefox.

Reply:
Oops... @LeoMcA suggested same formatting.